tests/apps/gaussian/gaussian.py

```diff
@@ -0,0 +1,64 @@
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+from reframe.core.backends import getlauncher
+import os
+
+
+class GaussianBaseTest(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        self.valid_prog_environs = ['builtin']
+        self.modules = ['Gaussian/g16_c01-avx2']
+
+        self.sanity_patterns = sn.assert_found(r' Normal termination of Gaussian',
+                                               self.stdout)
+        self.perf_patterns = {
+            'time': (
+                sn.extractsingle(
+                r'^real\t(?P<minutes>\S+)m\S+s',
+                'rfm_GaussianCPUTest_job.err', 'minutes', float) + 
+                sn.extractsingle(
+                r'^real\t\S+m(?P<seconds>\S+)s',
+                'rfm_GaussianCPUTest_job.err', 'seconds', float) / 60.0)
+        }
+
+        self.maintainers = ['Lewih']
+
+
+@rfm.simple_test
+class GaussianCPUTest(GaussianBaseTest):
+    def __init__(self):
+        super().__init__()
+        self.valid_systems = ['leibniz:single-node',
+                              'vaughan:single-node',
+                              'hydra:single-node']
+        self.reference = {
+            'leibniz:single-node': {
+                'time': (33.0, -0.5, 0.5, 'minutes/percentageOfMinute'),
```

Comment:
what do you mean with 'minutes/percentageOfMinute'? probably less confusing if we just use 'minutes'?

Reply:
Ah, now I remember , silly me. It is minutes.